meetings/2021-02-09-Community-Meeting.md

```diff
@@ -65,7 +65,7 @@ Name (#gluster-dev alias) - company
 
 ### GlusterFS - v10 and beyond
 
-* [Gluster X Backlog](https://docs.google.com/spreadsheets/d/1bMZdya4nDlqEo11IfCNrw4GcXUhof-Sw-YbbuzkjCmo/edit?usp=sharing) - Request to add any new features/big fix you are working on.
```

Comment:
Its good to have link for gluster 10 planned topics as community members other than RedHat can pitch in too. Why we are removing this?

Reply:
Because it's still under process and it for internal use.
A GitHub issue/tracker will soon be created by Saju and the same will be used to track the upstream contributions.

Reply:
Actually, Any upstream release discussion is not for internal use, community members should be involved in this.